File: Python/292_Nim_Game.py
import random


class Solution(object):
    def canWinNim(self, n):
        """
        :type n: int
        :rtype: bool
        """
        return n % 4 != 0


def test():
    sol = Solution()
    stone_num = random.randint(1, 1000)
    if sol.canWinNim(stone_num):
        print "When the number of stones are " + str(stone_num) + ", first win!"
    else:
        print "When the number of stones are " + str(stone_num) + ", second win!"


if __name__ == '__main__':
    test()

File: C#/292_Nim_Game.cs
using System;

namespace LeetCode
{
    public class Solution
    {
        public bool CanWinNim(int n)
        {
            return n % 4 != 0;
        }

        //Testcase
        static void Main(string[] args)
        {
            Solution sol = new Solution();
            int stone_num = new Random().Next(1, 1001);

            if (sol.CanWinNim(stone_num))
                Console.WriteLine("When the number of stones are " + stone_num + ", first win!");
            else
                Console.WriteLine("When the number of stones are " + stone_num + ", second win!");

        }
    }
}